cli/module_build.go

```diff
@@ -35,6 +39,41 @@ const (
 
 var moduleBuildPollingInterval = 2 * time.Second
 
+var apiKeyFlags = []cli.Flag{
+	&cli.StringFlag{
+		Name: loginFlagKeyIDVal,
```

Comment:
yes -- added it because I wanted there to be 'just one thing', but I'm not sure this is the thing

imo if we're multiplexing, we should set up a prefix so it's canonical + hard to break? like `kv:uuid:base64`?

Reply:
> I'm going to try viamClient / getapikeys approach, so this auth approach may go away

Talked a bit offline, and I agree with relying on existing auth approach/getting rid of this.

Reply:
using viamClient auth in 6d2b7758